assets/aml-benchmark/components/src/aml_benchmark/batch_output_formatter/main.py

```diff
@@ -89,13 +92,15 @@ def main(
     if ground_truth_input:
         input_file_paths = resolve_io_path(ground_truth_input)
         ground_truth_df = pd.DataFrame(read_jsonl_files(input_file_paths))
+        print(ground_truth_df)
```

Comment:
Can we use logger and make the print statement meaningful or delete this print statement?

Reply:
Yes. That was for debugging propose and will be deleted. Thank you for catching it!